catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -465,96 +508,115 @@ private boolean processBatch(CatalogFacade catalog, ArrayList<Metacard> metacard
     }
 
     private void moveToFailedIngestDirectory(File source) {
-        File destination = new File(failedIngestDirectory.getAbsolutePath() + File.separator + source.getName());
+        File destination = new File(
```

Comment:
I am a fan of `Paths.get()`.

`Paths.get(failedIngestDirectory.getAbsolutePath(), source.getName()).toFile()`

https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html#get(java.lang.String,%20java.lang.String...)